tools/reviewer/job/tasks/ReviewerMetadataUpdaterTask.java

```diff
@@ -153,5 +153,178 @@ public void run() {
   public Boolean shouldRun() {
     return !lock.isLocked();
   }
-}
 
+  public ReviewerConfig printStartupOsReviewerConfig() throws IOException {
+    String localRegistryFilePath = fileUtils.joinPaths(fileUtils.getCurrentWorkingDirectory(),
+        "reviewer_config.prototxt");
+    ReviewerConfig reviewerConfig = (ReviewerConfig) fileUtils.readPrototxt(localRegistryFilePath,
+        ReviewerConfig.newBuilder());
+    //System.out.println("StartupOs ReviewerConfig:\n" + reviewerConfig.toString());
+    return reviewerConfig;
+  }
+
+  public ReviewerConfig printHasadnaReviewerConfig() throws IOException {
```

Comment:
`printStartupOsReviewerConfig()` and `printHasadnaReviewerConfig()` methods are very similar. How about creating a universal method? The method can take as parameter a path to the file.

Reply:
Changed both to path getting methods, used by a universal method to get `reviewerConfig`s.